skglm/estimators.py

```diff
@@ -393,19 +342,15 @@ class Lasso(LinearModel, RegressorMixin):
     MCPRegression : Sparser regularization than L1 norm.
     """
 
-    def __init__(self, alpha=1., max_iter=100, max_epochs=50_000, p0=10,
-                 verbose=0, tol=1e-4, fit_intercept=True,
-                 warm_start=False, ws_strategy="subdiff"):
+    def __init__(self, alpha=1., tol=1e-4, max_iter=50, max_epochs=50_000,
+                 fit_intercept=True, warm_start=False):
         super().__init__()
+        self.alpha = alpha
         self.tol = tol
         self.max_iter = max_iter
+        self.max_epochs = max_epochs
         self.fit_intercept = fit_intercept
         self.warm_start = warm_start
-        self.verbose = verbose
```

Comment:
keep verbose, p0 and other attributes (otherwise they can't be set for this estimator)